docs/network/interfaces_and_networks.md

```diff
@@ -22,65 +22,54 @@ Reference](https://kubevirt.io/api-reference/master/definitions.html#_v1_network
 
 ## Networks
 
-Network backends are configured in `spec.networks`. A network must have
+Networks are configured in VMs `spec.template.spec.networks`. A network must have
 a unique name. Additional fields declare which logical or physical
 device the network relates to.
 
 Each network should declare its type by defining one of the following
 fields:
 
-<table>
-<colgroup>
-<col style="width: 50%" />
-<col style="width: 50%" />
-</colgroup>
-<thead>
-<tr class="header">
-<th>Type</th>
-<th>Description</th>
-</tr>
-</thead>
-<tbody>
-<tr class="odd">
-<td><p><code>pod</code></p></td>
-<td><p>Default Kubernetes network</p></td>
-</tr>
-<tr class="even">
-<td><p><code>multus</code></p></td>
-<td><p>Secondary network provided using Multus</p></td>
-</tr>
-</tbody>
-</table>
+| Type     | Description                                                                                  |
+|----------|----------------------------------------------------------------------------------------------|
+| `pod`    | Default Kubernetes network                                                                   |
+| `multus` | Secondary network provided using Multus or Primary network when Multus is defined as default |
 
 ### pod
 
-A `pod` network represents the default pod `eth0` interface configured
+Represents the default pod interface (typically `eth0`) configured
 by cluster network solution that is present in each pod.
+The main advantage of this network type is that it is native to Kubernetes, 
+allowing VMs to benefit from all network services provided by Kubernetes.
 
 ```yaml
-kind: VM
+# partial example - kept short for brevity 
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
 spec:
-  domain:
-    devices:
-      interfaces:
-        - name: default
-          masquerade: {}
-  networks:
-  - name: default
-    pod: {} # Stock pod network
+  template:
+    spec:
+      domain:
+        devices:
+          interfaces:
```

Comment:
It is very odd to see an example with fields that have not been explained yet.
IMO all `network` subtree examples should show only that tree (`networks` as root).

Later, after both networks and interfaces have been described and explained, full examples can be given.

If you agree, this can be done as a follow up.

Reply:
agreed